docs/RELEASING.md

```diff
@@ -0,0 +1,26 @@
+# Releasing
+
+The version scheme we use is **MAJOR.MINOR.PATCH** and stored in the [product_version](../product_version) file at the root of this repository.
+
+Open Crawler follows its own release versioning and does not follow the Elastic stack unified release schedule or versioning.
+
+## How to release
+
+Releasing is done entirely through Buildkite.
+The Open Crawler build can be found here: https://buildkite.com/elastic/crawler-docker-build-publish
```

Comment:
Is it good practice to add private links in the public repo? (I know it's convenient for us ...)

A few times when developing in ES I was told to not include private links

Reply:
I agree the links are not good practice. I copied how we do it for [Connectors](https://github.com/elastic/connectors/blob/579f72f170eb65d33c74344396576d865942b24b/docs/RELEASING.md) which has direct links to the buildkite pipeline, but two wrongs don't make a right.

I'll replace it with just the name of the buildkite job and a note at the top that this is intended for Elastic employees.